packages/xdl/src/__tests__/Project-test.ts

```diff
@@ -0,0 +1,49 @@
+import { ExpoConfig } from '@expo/config';
+import { AxiosRequestConfig } from 'axios';
+
+import { getSignedManifestStringAsync, getUnsignedManifestString } from '../Project';
+
+const mockManifest: ExpoConfig = {
+  name: 'Hello',
+  slug: 'hello-world',
+  owner: 'ownername',
+  version: '1.0.0',
+  platforms: ['ios'],
+};
+
+const mockSignedManifestResponse = JSON.stringify({
+  manifestString: JSON.stringify(mockManifest),
+  signature: 'SIGNATURE_HERE',
+  version: '1.0.0',
+});
+
+jest.mock('axios', () => ({
+  request: jest.fn(async (options: AxiosRequestConfig) => {
+    if (options.url.endsWith('/--/api/v2/manifest/sign')) {
+      expect(options.data.args).toEqual({
+        remoteUsername: mockManifest.owner,
+        remotePackageName: mockManifest.slug,
+      });
+      expect(options.data.manifest).toEqual(mockManifest);
```

Comment:
Hmmm, it seems a little weird to me that you're calling `expect` in a mock 🤔 I prefer to call `expect` only in test cases. Maybe `assert` would make more sense?

Reply:
Based on your feedback, I rewrote the mock so that the custom mock implementation is not needed at all and all the assertions are inside the test case. I think it made the test better, thanks!